### ISSUE TITLE
Feature Request:  Add metadata when saving visualize map to csv

### ISSUE THREAD

**Issue by sslupsky** [xLightsSequencer/xLights#2271]
When saving the visualize map to a csv file, the model names are saved but the other meta data shown on the screen is omitted.  It would be helpful to have the additional meta data in the csv file.

**Comment by keithsw1111** [comment]
Have you tried export models which has it all? I dont think we want to duplicate all the data again in the visualiser export.

**Comment by sslupsky** [comment]
Yes, I have been using the export models.  My observation regarding this issue was pointing out that when you mouse over the box in the visualizer, you can see some of the metadata but this doesn't appear csv when you export.

**Comment by keithsw1111** [comment]
And ..., this just seems like a feature for the sake of a feature ... what problem are you trying to solve?

**Comment by sslupsky** [comment]
There is no problem.  I am making an suggestion regarding how to improve the usefulness of the visualizer print.  I would like to print that page(s) and put it in my back pocket when I go to the garage to work on the layout.  Without the metadata, i have to use my spreadsheet which does not have the intuitive layout that the visualizer has.

**Comment by sslupsky** [comment]
I will. Lose this now.  It is not important or urgent so is easily pruned.

**Comment by computergeek1507** [comment]
I plan on improving this

**Comment by cybercop23** [comment]
@computergeek1507 shoudl this still be open? mabye the 3d viz thing you were working on?

**Comment by derwin12** [comment]
Scott appears to have done this in 2021.
